plugins/jobs/src/js/components/JobSearchFilter.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Created JIRA to remove those comments https://jira.mesosphere.com/browse/DCOS-38200